package.json

```diff
@@ -144,6 +144,7 @@
 		"@automattic/webpack-inline-constant-exports-plugin": "file:./packages/webpack-inline-constant-exports-plugin",
 		"@babel/cli": "7.8.3",
 		"@babel/core": "7.8.3",
+		"@jameslnewell/react-promise": "^3.3.0",
```

Comment:
This is not a `devDependency`, is it? It should be defined in `packages/media-library/package.json` as `dependency`.